#!/usr/bin/env python

PACKAGE = "rbx1_nav"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("test_angle", double_t, 0, "Test Angle", 360.0, 0.0, 360.0)

gen.add("speed", double_t, 0, "Angular speed in radians per second", 0.5, -1.5, 1.5)

gen.add("tolerance", double_t, 0, "Error tolerance to goal angle in degrees", 1.0, 1.0, 10.0)

gen.add("odom_angular_scale_correction", double_t, 0, "Angular correction factor", 1.0, 0.0, 3.0)

gen.add("start_test", bool_t, 0, "Check to start the test", False)


exit(gen.generate(PACKAGE, "calibrate_angular", "CalibrateAngular"))
